www/apps/book/app/basics/modules/page.mdx

```diff
@@ -0,0 +1,272 @@
+export const metadata = {
+  title: `${pageNumber} Modules`,
+}
+
+# {metadata.title}
+
+In this chapter, you’ll learn about modules and how to create them.
+
+## What is a Module?
+
+A module is a package of reusable commerce or architectural functionalities. These functionalities are implemented within the module in a class called a service.
+
+In the module, you can create new tables in the database. Then, you manage their records using the module's service, which connects to the database.
+
+A module is integrated as a building block in your Medusa application, without implications on the existing setup.
+
+<Note title="Tip">
+
+Modules and services have other uses too, which you'll learn about in later chapters.
+
+</Note>
+
+---
+
+## How to Create a Module?
+
+Modules are created in a sub-directory of `src/modules`.
+
+For example, create the directory `src/modules/hello`.
+
+### 1. Create Data Model
+
+A data model represents a table in the database. It's created in a TypeScript or JavaScript file under the module's `models` directory.
+
```

Comment:
thought: feel like there is a need for setting the agenda for the steps that are coming. Just one sentence on the high-level objective here would be helpful.

E.g.: "In the following steps we will create a data model called `MyCustom`, we will then add a ModuleService to manage that data model, and finally we will use the module in an API route."